data/json/construction.json

```
@@ -2915,5 +2915,16 @@
     "qualities": [ { "id": "DIG", "level": 2 } ],
     "pre_flags": [ "DIGGABLE", "FLAT" ],
     "post_terrain": "t_pit"
+  },
+  {
+    "type": "construction",
+    "description": "Build Arc Furnace",
+    "category": "FURN",
+    "required_skills": [ [ "fabrication", 3 ] ],
+    "time": "120 m",
+    "qualities": [ [ { "id": "HAMMER", "level": 2 } ], [ { "id": "SAW_M", "level": 1 } ], [ { "id": "WELD", "level": 2 } ], [ { "id": "DRILLL", "level": 3 } ], [ { "id": "WRENCH", "level": 2 } ] ],
```

```
{ "id": "DRIL", "level": 1 }
{ "id": "DRILL", "level": 2 }
{ "id": "DRILLL", "level": 3 }
```
sorry...

---

Humor or a legit change I need to make?

---

There is a legit change
```suggestion
    "qualities": [ [ { "id": "HAMMER", "level": 2 } ], [ { "id": "SAW_M", "level": 1 } ], [ { "id": "WELD", "level": 2 } ], [ { "id": "DRILL", "level": 3 } ], [ { "id": "WRENCH", "level": 2 } ] ],
```

---

oh man my sticky keys.  Thanks!